cps/kni.c

```diff
@@ -344,16 +344,16 @@ kni_config_link(struct rte_kni *kni)
 }
 
 void
-route_event_sock_close(struct cps_config *cps_conf)
+rd_event_sock_close(struct cps_config *cps_conf)
```

Comment:
I'm fine with the prefix "rd_" to mean "routing daemon" in the code, but we should have a comment explaining the prefix because "rd" reminds me more of "read".